pom.xml

```diff
@@ -25,6 +25,13 @@
     <tag>HEAD</tag>
   </scm>
 
+  <repositories>
```

Comment:
I suggest you replace it with the Maven Central repo. Because the JCenter repo is [deprecated and read-only right now](https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/)

Reply:
i replaced it with https://repo1.maven.org/maven2